tests/subsys/fs/littlefs/prj.conf

```diff
@@ -6,9 +6,6 @@ CONFIG_MAIN_STACK_SIZE=4096
 # Performance tests need custom buffer allocation
 CONFIG_MINIMAL_LIBC_MALLOC_ARENA_SIZE=8192
 
-# Configuration tests require variable cache sizes
-CONFIG_FS_LITTLEFS_FC_HEAP_SIZE=16384
```

Comment:
this was modified in previous commit

Reply:
Yes.  In the previous commit the existing `prj.conf` line needed to be changed from using the mempool-based customization Kconfig variable to the equivalent heap-based variable; this targeted all littlefs tests.

In this commit the configuration moves from `prj.conf` to `testcase.yaml` because it's not common to all test configurations.

I don't think any other way of doing it is clear and bisectable.